packages/vue3-vite/src/generators/vue3-vite/files/src/app/App.vue.template

```diff
@@ -1,12 +1,14 @@
+<% const $assetsPath = useLocalAlias ? '@assets' : '../assets' %>
```

Comment:
Since the app is using either "local alias" or "global alias", we need a way to change the path of files in these template files.
- If local alias is used, all imports in the app will look like `@app/stuff` or `@assets/stuff`.
- If global alias is used, all imports in the app will be relative, such as `../../stuff' or '../../assets/stuff'.

One way to do it is to put these values in the options used when calling `generateFiles()`, but because not all files are in the same hierarchy, we'll have to put a lot of different strings in `templateOptions`, such as `..`, `../../`, `../../../`, which doesn't sound very good.

So instead I just declare these variables in each individual template files. For some reason I think there should be a better solution. 🤔

Reply:
Also, in order to avoid any conflict, I use a dollar sign as the prefix of these variables.
I think it's not very intuitive.

Reply:
It's a reasonable solution, but makes it a bit difficult to change the path names, since all the template files using them would have to be adjusted.

I changed it to use functions passed to the template, it's not a perfect solution but at least the path names are defined in one place:

```ts
function makeAssetPath(useLocalAlias: boolean) {
  return (localRoot: string) =>
    useLocalAlias ? '@assets' : `${localRoot}/assets`;
}

function makeAppPath(useLocalAlias: boolean) {
  return (localRoot: string) => (useLocalAlias ? '@app' : localRoot);
}
```